mongodb/kubedb-mongodb-ops-request-failed.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.appscode.com/v1alpha1
+kind: Runbook
+metadata:
+  namespace: monitoring
+  name: kubedb-mongodb-ops-request-failed
+spec:
+  diagnosticChecks:
+    - type: InspectPodConditions
+      args:
+        description: Describe the MongoDBOpsRequest and check the conditions.
```

Comment:
can we do something similar for conditions? Perhaps we could have a generic `InspectConditions` type?